python/tskit/trees.py

```diff
@@ -2893,6 +2893,20 @@ def sackin_index(self):
         """
         return self._ll_tree.get_sackin_index()
 
+    def num_lineages(self, t):
+        """
+        Returns the number of lineages present in this tree at time ``t``. This
+        is defined as the number of branches in this tree (reachable from roots)
+        that intersect with ``t``, plus the number of roots with time ``>= t``.
```

Comment:
shouldn't this be `<= t`?

Reply:
Also, I'd say explicitly that if a node with three children has time `t` then it will count as one lineage, not three.